obsdfw/Makefile

```diff
@@ -92,6 +92,9 @@ obsdfw: \
 /etc/miniupnpd.conf : miniupnpd.conf
 	install -o root -g wheel -m 0644 \
 	  miniupnpd.conf /etc/miniupnpd.conf
+	pkg_add miniupnpd
```

Comment:
This definitely needs to move above the install, or the default config will eat yours the first time.
I think it's also wise to make this conditional, something like:
`@if ! [ -d /var/db/pkg/miniupnpd* ]; then pkg_add miniupnpd; fi`

Reply:
lol fyi I was just wayyyy to lazy to figure out A) conditionals in Make and 2) How to determine if it was already installed so tyvm

Reply:
I think the @ will work to just shell out